src/drivers/amd/renoir/acp_dma.c

```diff
@@ -201,8 +201,11 @@ int dma_setup(struct dma_chan_data *channel,
 			} else {
 				/* Capture */
 				dma_cfg->base = dma_config_dscr[dscr_strt_idx].src_addr;
```

Comment:
I think you should split this into 2 patches. 

First one modifying acp_dma.c with a proper explanation of why the change is needed. Second patch actually enabling CONFIG_PROBE and friends.